src/FormBuilder/resources/js/form-builder/src/components/ClassicEditor/index.js

```diff
@@ -37,7 +37,7 @@ function isTmceEmpty(editor) {
 /**
  * @since 3.2.0
  */
-export default function ClassicEditor({id, label, content, setContent, rows = 20}) {
+export default function ClassicEditor({id, label = null, content, setContent, rows = 20}) {
```

Comment:
Resolved: c5640469ce8d6e87b13b9e4f95a8375131ba91af

Reply:
@pauloiankoski Sorry I mispose - we actually [have a PR](https://github.com/impress-org/givewp/pull/7122) that is working to remove this component.  Right now this is technically being used through the window on a couple add-on develop branches and want to avoid that change here to prevent any side effects.  

So for this PR[ we can keep this](https://github.com/impress-org/givewp/commit/c5640469ce8d6e87b13b9e4f95a8375131ba91af) - maybe even just assign the window object it to the form-buider-libary component

Reply:
Resolved: 6596c88462d0ea9dcfb8b424af6b41b26c43d63d